recipes/monitor-server-api/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "monitor-server-api" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "4ef28cf26db464e65160b6672650d36de0cceaae9004777bd2d238a712e2785f" %}
+{% set md5 = "246a15525193fb6b3618afc4c151335f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  md5: {{ md5 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build: []
```

Comment:
You can drop the `build` section

Reply:
done

Reply:
Thanks for spotting the badge issue.  🥇 
Regarding the LICENSE point you raised, I wasn't aware of this "trick". Thanks for point it out 👍🏻